egs2/babel/asr1/RESULTS.md

```diff
@@ -0,0 +1,25 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# RESULTS
+## Environments
+- date: `Sun Mar  1 21:23:27 EST 2020`
+- python version: `3.7.6 (default, Jan  8 2020, 19:59:22)  [GCC 7.3.0]`
+- espnet version: `espnet 0.6.0`
+- pytorch version: `pytorch 1.4.0`
```

Comment:
Could you merge this file into README.md? I'd like to show the result more easy.